groups/prometheus/module-prometheus/variables.tf

```diff
@@ -28,9 +28,9 @@ variable "prometheus_metrics_port" {
   description = "The metrics port to be used"
 }
 
-variable "prometheus_web_fqdn" {
+variable "tag_name_regex" {
   type = string
-  description = "The Prometheus target to be used, such as Concourse"
+  description = "The Prometheus tag to be used for discovering EC2 instances"
```

Comment:
This description will need updating now too as it no longer matches. I'd suggest something like `The tag name regex used to discover EC2 instances`

Reply:
This still needs addressing